packages/typescript-estree/src/parser-options.ts

```diff
@@ -123,6 +123,11 @@ export interface ProjectServiceOptions {
    * @default 8
    */
   maximumDefaultProjectFileMatchCount_THIS_WILL_SLOW_DOWN_LINTING?: number;
+
+  /**
+   * Additional file extensions which should be considered in the TypeScript Program compilation.
+   */
+  extraFileExtensions?: string[];
```

Comment:
[Docs] Nit: would you mind putting these options in alphabetical order please? It helps keep the page well structured & easy to navigate.

Reply:
I had another look at this, and realised this must've been a left-over from a previous iteration, this change is actually not needed, since we rely on the extraFileExtensions of the parent object (same as the non-project-service setup).